composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "friendsofsymfony/rest-routing-bundle",
+    "type": "symfony-bundle",
+    "description": "This Bundle automatic route registration for the Controllers",
+    "keywords": ["rest"],
+    "license": "MIT",
+    "autoload":{
+        "psr-4":{
+            "FOS\\RestRoutingBundle\\": ""
+        },
+        "exclude-from-classmap": [
+            "Resources/",
+            "Tests/"
+        ]
+    },
+    "require": {
+        "php": "^7.2",
+        "doctrine/inflector": "^1.0",
+        "friendsofsymfony/rest-bundle": "^3.0@dev",
```

Comment:
I would change this to `^3.0` and add `minimum-stability` to `dev` instead. So you will not have to change this in the future.